modules/olm-terms.adoc

```diff
@@ -2,32 +2,35 @@
 //
 // * operators/understanding/olm/olm-common-terms.adoc
 
+[id="olm-common-terms-glossary_{context}"]
+= Common Operator Framework terms
```

Comment:
@adellape You can't have more than one level 1 heading (= Heading) in a file, so this was my attempt at fixing this file. Can you take a look and see if this is okay how I added this, or if you have a different way/text you'd rather see?

Preview: https://fixin-build-issues--ocpdocs.netlify.app/openshift-enterprise/latest/operators/understanding/olm-common-terms.html#olm-common-terms-glossary_olm-common-terms

Reply:
Yes, that works. Thanks!